rclcpp/include/rclcpp/client.hpp

```diff
@@ -470,12 +473,12 @@ class Client : public ClientBase
   : ClientBase(node_base, node_graph)
   {
     using rosidl_typesupport_cpp::get_service_type_support_handle;
-    auto service_type_support_handle =
+    service_type_support_handle_ =
```

Comment:
I think you can put this in the initializer list now.

Reply:
Good call, fixed in 01aa7e0